2022-11-03 11:14:51,289 Converting data/part268/in/kerala/2016/2016_134_100.pdf to img...
2022-11-03 11:15:00,077 Parsing first and last page data..
2022-11-03 11:15:13,426 Detecting and parsing data/part268/in/kerala/2016/2016_134_100.pdf boxes..
2022-11-03 11:15:13,427 Processing page 3
2022-11-03 11:15:52,245 Processing page 4
2022-11-03 11:16:31,425 Processing page 5
2022-11-03 11:17:11,034 Processing page 6
2022-11-03 11:17:50,592 Processing page 7
2022-11-03 11:18:28,332 Processing page 8
2022-11-03 11:19:07,021 Processing page 9
--- Logging error ---
Traceback (most recent call last):
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/logging/__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
UnicodeEncodeError: 'latin-1' codec can't encode character '\u2019' in position 64: ordinal not in range(256)
Call stack:
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/concurrent/futures/thread.py", line 80, in _worker
    work_item.run()
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "../parse_unsearchable_rolls/parser/parser.py", line 320, in get_data
    processed_box = self.process_boxes_text(text)
  File "../parse_unsearchable_rolls/parser/parser.py", line 149, in process_boxes_text
    result, last_key, is_splitted = self.columns_split(r, self.columns, result, last_key)
  File "../parse_unsearchable_rolls/parser/parser.py", line 255, in columns_split
    result, last_key, is_splitted = self.handle_separator_without_column(r, result, last_key)
  File "../parse_unsearchable_rolls/parser/parser.py", line 226, in handle_separator_without_column
    logging.warning(f'Split uncaught {r} {result}')
Message: "Split uncaught ollgyeaie Broenié&> aeons\u2019 OrderedDict([('number', '189'), ('id', 'FVM1080514'), ('\u0d2a\u0d47\u0d30\u0d4d', '\u0d30\u0d3e\u0d39\u0d41\u0d32\u0d4d\\u200d \u0d0e\u0d38\u0d4d\\u200c \u0d0f\u0d32\u0d4d\\u200d'), ('\u0d05\u0d1a\u0d4d\u0d1b\u0d28\u0d4d\u0d31\u0d46 \u0d2a\u0d47\u0d30\u0d4d', '\u0d36\u0d4d\u0d30\u0d40\u0d15\u0d23\u0d4d\u0d20\u0d28\u0d4d\\u200d \u0d28\u0d3e\u0d2f\u0d30\u0d4d\\u200d'), ('\u0d35\u0d40\u0d1f\u0d4d\u0d1f\u0d41\u0d28\u0d2e\u0d4d\u0d2a\u0d30\u0d4d', '20/830 (0/)')])"
Arguments: ()
2022-11-03 11:19:40,104 Split uncaught ollgyeaie Broenié&> aeons\u2019 OrderedDict([('number', '189'), ('id', 'FVM1080514'), ('\u0d2a\u0d47\u0d30\u0d4d', '\u0d30\u0d3e\u0d39\u0d41\u0d32\u0d4d\u200d \u0d0e\u0d38\u0d4d\u200c \u0d0f\u0d32\u0d4d\u200d'), ('\u0d05\u0d1a\u0d4d\u0d1b\u0d28\u0d4d\u0d31\u0d46 \u0d2a\u0d47\u0d30\u0d4d', '\u0d36\u0d4d\u0d30\u0d40\u0d15\u0d23\u0d4d\u0d20\u0d28\u0d4d\u200d \u0d28\u0d3e\u0d2f\u0d30\u0d4d\u200d'), ('\u0d35\u0d40\u0d1f\u0d4d\u0d1f\u0d41\u0d28\u0d2e\u0d4d\u0d2a\u0d30\u0d4d', '20/830 (0/)')])
2022-11-03 11:19:45,451 Processing page 10
2022-11-03 11:20:23,069 Processing page 11
2022-11-03 11:21:01,858 Processing page 12
2022-11-03 11:21:40,050 Processing page 13
2022-11-03 11:22:16,888 Processing page 14
2022-11-03 11:22:56,179 Processing page 15
2022-11-03 11:23:33,761 Processing page 16
2022-11-03 11:24:12,125 Processing page 17
2022-11-03 11:24:51,391 Processing page 18
2022-11-03 11:25:29,506 Processing page 19
2022-11-03 11:26:07,850 Processing page 20
2022-11-03 11:26:45,852 Processing page 21
2022-11-03 11:27:22,278 Processing page 22
2022-11-03 11:28:00,909 Processing page 23
2022-11-03 11:28:38,209 Processing page 24
2022-11-03 11:29:17,064 Processing page 25
2022-11-03 11:29:54,967 Processing page 26
2022-11-03 11:30:32,309 Processing page 27
2022-11-03 11:31:10,277 Processing page 28
2022-11-03 11:31:43,281 Processing page 29
--- Logging error ---
Traceback (most recent call last):
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/logging/__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
UnicodeEncodeError: 'latin-1' codec can't encode characters in position 129-132: ordinal not in range(256)
Call stack:
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/concurrent/futures/thread.py", line 80, in _worker
    work_item.run()
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "../parse_unsearchable_rolls/parser/parser.py", line 320, in get_data
    processed_box = self.process_boxes_text(text)
  File "../parse_unsearchable_rolls/parser/parser.py", line 149, in process_boxes_text
    result, last_key, is_splitted = self.columns_split(r, self.columns, result, last_key)
  File "../parse_unsearchable_rolls/parser/parser.py", line 255, in columns_split
    result, last_key, is_splitted = self.handle_separator_without_column(r, result, last_key)
  File "../parse_unsearchable_rolls/parser/parser.py", line 226, in handle_separator_without_column
    logging.warning(f'Split uncaught {r} {result}')
Message: "Split uncaught algynmia 20/1006 = (0/1372) OrderedDict([('number', '793'), ('id', 'KL/20/136/198565'), ('\u0d2a\u0d47\u0d30\u0d4d', '7]')])"
Arguments: ()
2022-11-03 11:32:02,994 Split uncaught algynmia 20/1006 = (0/1372) OrderedDict([('number', '793'), ('id', 'KL/20/136/198565'), ('\u0d2a\u0d47\u0d30\u0d4d', '7]')])
2022-11-03 11:32:11,272 Processing page 30
2022-11-03 11:32:48,519 Processing page 31
2022-11-03 11:33:25,246 Processing page 32
2022-11-03 11:34:03,680 Processing page 33
2022-11-03 11:34:43,315 Processing page 34
2022-11-03 11:35:20,589 Processing page 35
2022-11-03 11:35:57,889 Processing page 36
2022-11-03 11:36:35,648 Processing page 37
2022-11-03 11:37:14,299 Processing page 38
2022-11-03 11:37:52,237 Processing page 39
2022-11-03 11:38:29,772 Processing page 40
2022-11-03 11:39:07,221 Processing page 41
2022-11-03 11:39:45,465 Processing page 42
2022-11-03 11:40:23,691 Processing page 43
/share/svasudevan2lab/gs-delhi/parse_unsearchable_rolls/scripts/kerala/joblist_2016/kerala268.sh: line 1: 28404 Killed                  python scripts/kerala/kerala_2016.py data/part268/

#################
Epilogue Job Info:
Job ID: 466350[268].acropolis
User ID: t-9adar
Job Name: Kerala2016JobArray-268
Resource List: nodes=1:ppn=1,mem=2gb,neednodes=1:ppn=1
Resources Used: cput=00:24:41,vmem=3397300kb,walltime=00:26:05,mem=2007888kb,energy_used=0
Exit Code: 137
Exit code 137 job terminated, resource limit violation
Generally this means you need to request more RAM or CPU
Please compare your resources used/requested
#################

